HHS/Head-Start-TTADP
# [TTAHUB-509] Add screenshot button to widgets

## Description of change
Use html2canvas to add a screenshot button to the topic/frequency graph and the TTA hours widget. The topic/frequency graph still has some issues; namely, there is some gray artifacts at the bottom and, more obviously, it is enormous. The first I don't think is easily solvable, the second is by request/design.

## How to test
You'll have to install the new dependencies. Ensure that you can get a screenshot of the widgets in question.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-509

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
